etc/edge.router.tmpl.yml

```diff
@@ -0,0 +1,90 @@
+v: 3
```

Comment:
Interesting. I have a number of scripts that make use of template files just like this - just outside of the repo. 

Would it make sense to:
- move this to `etc/tmpl` as to not clutter this directory
- parameterize most settings

?

Reply:
sure. The controller host/ip could be templated. Anything else you'd want to templatize?

Reply:
We really need some sort of namespaced+`FABLAB_TEMPLATE_PATH'd template file resolution. If we had a `FABLAB_TEMPLATE_PATH` variable with `:` separated entries, people could put that stuff wherever they want.

Reply:
Wrong project. Disregard. 💯

Reply:
Moved and templated controller host/port